src/ecredis.erl

```diff
@@ -577,6 +581,18 @@ group_by_destination(#query{command = Command,
     end.
 
 
+try_eredis_query(Pid, Command) ->
+    try
+        eredis_query(Pid, Command)
+    catch
+        exit : {timeout, St} ->
+            ?ERROR("eredis genserver timeout Pid: ~p Commands: ~p ~p", [Pid, Command, St]),
+            {error, gen_server_timeout};
```

Comment:
we should call it just timeout?

Reply:
Ok, lets call it just timeout. I thought it is interesting if it comes from the gen_server or maybe the socket itself.